kpi/serializers/v2/service_usage.py

```diff
@@ -265,12 +270,12 @@ def _get_organization_details(self, organization_id=None):
         if not organization_id:
             return
 
-        organization = Organization.objects.filter(
+        self._organization = Organization.objects.filter(
```

Comment:
I think we can keep `organization` as it only lives within the scope of `_get_organization_details()`. 
Any reasons to make it a protected attribute of the class?